libs/docs/platform/table/examples/platform-table-responsive-columns-example.component.html

```diff
@@ -16,7 +16,7 @@
     [rowsActivable]="true"
     emptyTableMessage="No data found"
     [noBorderX]="true"
```

Comment:
not related to the issue, but since we are changing `noBorderY` to `noVerticalBorders`, shouldn't we also change `noBorderX` to `noHorizontalBorders`?

Reply:
They actually do different things. Input names could use some improvement, but the API docs clarify it a bit.

```
    /** Table without horizontal borders. */
    @Input()
    noHorizontalBorders = false;
```

vs

```
    /** Table body without horizontal borders. */
    @Input()
    noBorderX = false;
```

noHorizontalBorders:
![Screenshot 2025-05-20 at 5 44 20 PM](https://github.com/user-attachments/assets/10f2c89a-2136-44e1-9ede-37f9b25597eb)


noBorderX:
![Screenshot 2025-05-20 at 5 42 17 PM](https://github.com/user-attachments/assets/1b88518c-7847-4785-992a-b7458f68358a)